### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by msandi** [johngodley/redirection#853]
I tried to create a new redirect and received this error

I'm trying to have this:
http://dykenpond.org/events
redirect to this:
http://dykenpond.org/?page_id=3134

For now I'm going to create my own redirect, let me know when you have this resolved.
Thanks!
```
Plugin: 3.2
WordPress: 4.9.4 (single)
PHP: 7.1.14
Browser: Mozilla/5.0 (Windows NT 6.1; WOW64; rv:59.0) Gecko/20100101 Firefox/59.0
JavaScript: http://www.friendsofdykenpond.org/dykenpond/wp-content/plugins/redirection/redirection.js
REST API: http://dykenpond.org/index.php?rest_route=/

Error: NetworkError when attempting to fetch resource.
```

**Comment by johngodley** [comment]
You have two URLs there - www.friendsofdykenpond.org and dykenpond.org. What are you URLs in Settings > General?

**Comment by msandi** [comment]
Hi:
I was trying to use dykenpond.org  But since it's a wordpress site, I am unable to create a page dykenpond.org/events

But at first your redirection plugin wasn't even letting me create the plugin, even though I knew I had created one previously.  I had to change to "Forms" to get it to work at all.

Then I created a redirection page on my www.friendsofdykenpond.org  site to direct to the wordpress page and also added this redirection in your plugin for added assurance.  Now it's working.
